sample_code/ep420/rev02/__init__.py

```diff
@@ -0,0 +1,2 @@
+import pkg_resourses
+pkg_resourses.declare_namespace(__name__)
```

Comment:
it's `pkg_resources` I believe!

Reply:
lol, thanks, I forgot to do `add -u` (even tested it beforehand)🤦